packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -163,44 +188,59 @@ function getPrismaClient(hasGenerated = false): {
   return localPrisma
 }
 
-function getPluginConfig() {
+function getPrismaModels() {
   // Extract the models from the prisma client and use those to
   // set up internal redirects for the return values in resolvers.
   const localPrisma = getPrismaClient()
   const prismaModels = localPrisma.ModelName
 
-  Object.keys(prismaModels).forEach((key) => {
-    prismaModels[key] = `@prisma/client#${key} as Prisma${key}`
-  })
-
   // This isn't really something you'd put in the GraphQL API, so
   // we can skip the model.
   if (prismaModels.RW_DataMigration) {
     delete prismaModels.RW_DataMigration
   }
 
+  return prismaModels
+}
+
+function getPluginConfig(side: CodegenSide) {
+  const prismaModels: Record<string, string> = getPrismaModels()
+  try {
```

Comment:
This try ... catch isn't actually wrapping anything that can throw, is it?

It used to wrap `prismaModels.RW_DataMigration`, where `prismaModels` could be `undefined`. But that's not the case anymore. Now it's guaranteed to be a Record.

Reply:
True!

Reply:
Removed!